File: figureGeometric - copia/circle.java
public class circle extends figureGeometric {
    private final double radius;
    private final double pi;
    private double calculate;

    public circle(double radius) {
        super("circle", "circulo");
        this.radius = radius;
        this.pi = 3.1416;
    }

    public double getRadius() {
        return this.radius;
    }

    public double getPi() {
        return this.pi;
    }

    @Override
    public double calculateArea() {
        this.calculate= this.getRadius()*this.getRadius()*this.getPi();
        return this.calculate;
    }
}

File: figureGeometricphp - copia/php/circle.php
<?php


class Circle extends FigureGeometric {
    private ?float $radius;
   
    private const PI = 3.1416;
    private ?float $calculate = null;

    public function __construct(float $radius) {
        parent::__construct("circle", "círculo");
        $this->radius = $radius;
    }

    public function getRadius(): ?float {
        return $this->radius;
    }

    
    public function getPi(): float {
        return self::PI;  
    }

    public function getCalculateArea(): ?float {
        $this->calculate = ($this->getRadius() * $this->getRadius() * self::PI);
        return $this->calculate;
    }
}
?>
